### DISCUSSION TITLE
Request for feedback from release-plz-author

### DISCUSSION THREAD

**Discussion by marcoieni** [polarmutex/beancount-language-server#360] [General]
Hi, I'm the [release-plz](https://github.com/MarcoIeni/release-plz) author. 👋 
I noticed that in https://github.com/polarmutex/beancount-language-server/commit/7b88e5c9557d2a690f24c15edb08c05fae49cbb3 you stopped using release-plz. Can you tell me why, so that I can make release-plz better? 🙏 

--- 

I hope this message is not considered spam. This was the least intrusive way I found to contact you.
If there's a better place to discuss this, please tell me and delete this message.

**Comment by polarmutex** [comment]
If I remember correctly, I had trouble or was not simple combining it with cargo-dist. I needed something fast, I am willing to try again when I have more free time

**Reply by marcoieni** [reply]
Thank you for answering. 🙏 

> I needed something fast, I am willing to try again when I have more free time

That's totally understandable :)

I haven't tried combining release-plz with cargo-dist, but I successfully used release-plz with the [upload-rust-binary-action](https://github.com/taiki-e/upload-rust-binary-action), which is similar.
I documented this [here](https://marcoieni.github.io/release-plz/extra/releasing-binaries.html).

I think release-plz and cargo-dist complement each other: release-plz creates the tag, while cargo-dist releases the binary!
Please, let me know how the integration between release-plz and cargo-dist goes for you, I'm very curious and I'm happy to help :)
